lib/WebpackOptionsValidationError.js

```diff
@@ -116,7 +116,7 @@ class WebpackOptionsValidationError extends WebpackError {
 						if (!required.includes(property)) return property + "?";
 						return property;
 					})
-					.concat(schema.additionalProperties ? ["..."] : [])
+					.concat(schema.additionalProperties ? ["…"] : [])
```

Comment:
This will break

Reply:
Only if someone relies on error messages which is absolutely wrong.